clients/js/test/revisions/v1/additionalSigner.test.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable prefer-template */
+import { generateSigner } from '@metaplex-foundation/umi';
+import test from 'ava';
+import { RuleSetRevisionV1, isAdditionalSignerRuleV1 } from '../../../src';
+import { createUmiSync } from '../../_setup';
+
+test('isAdditionalSigner', async (t) => {
+  const umi = createUmiSync();
+  const owner = generateSigner(umi).publicKey;
+  const publicKeyA = generateSigner(umi).publicKey;
+  const revision: RuleSetRevisionV1 = {
+    libVersion: 1,
+    ruleSetName: 'My Rule Set',
+    owner: [...owner.bytes],
+    operations: {
+      deposit: {
+        AdditionalSigner: {
+          account: [...publicKeyA.bytes],
+        },
+      },
+    },
+  };
+
+  t.is(isAdditionalSignerRuleV1(revision.operations.deposit), true);
```

Comment:
Ava actually has a `t.true` method. Same for other tests.

```suggestion
  t.true(isAdditionalSignerRuleV1(revision.operations.deposit));
```